source/localizable/models/relationships.md

```diff
@@ -43,7 +43,7 @@ import Model from 'ember-data/model';
 import { belongsTo } from 'ember-data/relationships';
 
 export default Model.extend({
-  post: belongsTo('post')
+  blogPost: belongsTo('blog-post')
```

Comment:
If we change this to `blogPost` from `post`, then we may want to rename all other cases of `blogPost` on this page, which are a ton ...